package.json

```diff
@@ -58,7 +58,14 @@
     "webpack-dev-server": "^3.5.1"
   },
   "dependencies": {
+    "axios": "^0.19.0",
+    "dotenv": "^8.0.0",
```

Comment:
Read the comment below. This should be a dev dependency. If not, then it can stay here.